.github/workflows/lint.yaml

```diff
@@ -0,0 +1,30 @@
+name: Linting
```

Comment:
Also add the bytecode publish workflow to create releases. And if we need to publish to npm, can add that action too.